src/context/CollectionContext/index.js

```diff
@@ -25,22 +31,14 @@ const collectionGrapgqlQuery = (id) => {
           level
           objectFileGroupId
           collectionId
-          defaultFilePath
-          partiallyDigitized
-          files {
-            items {
-              id
-              label
-              objectFileGroupId
-            }
+          defaultFile {
+            iiifImageUri
+            filePath
```

Comment:
Do you also want to pull iiifImageServiceUri here to use this instead of a hard coded base uri elsewhere?

Reply:
done.